packages/app/src/app/components/CreateNewSandbox/CreateSandbox/SearchBox/SearchBox.tsx

```diff
@@ -12,7 +12,7 @@ type SearchProps = {
 export const SearchBox = ({
   value,
   onChange,
-  placeholder = 'Search',
+  placeholder = 'Search Templates',
```

Comment:
The previous search input had an icon with a `/` to indicate that pressing `/` key focuses the search bar. Should we re-include that?